usaspending_api/api_docs/api_documentation/download/custom_award_data_download.md

```diff
@@ -0,0 +1,81 @@
+## [Custom Award Data Download](#Custom_Award_Data_Download)
+**Route:** `/v2/bulk_download/awards/`
```

Comment:
It's inconsistent between files whether `/api` is included in the route

Reply:
Yea, we had different writers, which led to little things like this!  Nice catch